### PR TITLE
Uyuni 2024.07

### PR DESCRIPTION
**Pull request by raulillo82**
- [x] Add missing release notes
- [x] Update the pdf docs in case they get changed or any fixes (docs working on it)
- [x] Check that release date is the expected one (as of today, July ~12th~ 19th)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -235,20 +235,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">          
-              <span>15/05/2024</span>
+              <span>12/07/2024</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2024.05 is now available removing technology preview from Server/Proxy container images, native support for AppStream repositories, automatic migration to the Salt Bundle and more! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.07 is now available with new products enabled, changed behaviour of repo-sync Confidential Computing Attestation UI and more! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
Comment (juliogonzalez) [inline index.html:241]:
```suggestion
              <p>Uyuni 2024.07 is now available with new products enabled, changed behaviour of repo-sync, Confidential Computing Attestation UI and more! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

File: pages/news.html [hunk 2]
```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+	  <p><strong>12/07/2024</strong> - Uyuni 2024.07 is now available with new products enabled, changed behaviour of repo-sync Confidential Computing Attestation UI and more! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
Comment (juliogonzalez) [inline pages/news.html:91]:
```suggestion
	  <p><strong>12/07/2024</strong> - Uyuni 2024.07 is now available with new products enabled, changed behaviour of repo-sync, Confidential Computing Attestation UI and more! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

**Comment by juliogonzalez** [comment]
Please add the list with checkboxes of things pending for this PR, at its description.

From the top of my mind, final release notes, and final (and real) release date.

**Review by deneb-alpha — APPROVED** [review]
ready to  release

**Pull request merged**